mslib/mscolab/server.py

```diff
@@ -197,6 +197,11 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
+def allowed_file(filename):
+    """ Check if the uploaded file is in the allowed set of extensions """
```

Comment:
when we store the file as image on the server by the server this not needed, when we want to check incoming files of those types that is not enough, because when you add to a pdf a different extensions it wil gets over this.